src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logoLink {
+  margin: 10px 0;
+  text-decoration: none;
+}
+
+.navList {
+  padding: 0;
+  display: flex;
+  margin: 0;
+}
+
+.navItem {
+  display: flex;
+  position: relative;
+}
+
+.navLink {
+  padding: 23px 0;
+  font-size: 12px;
+  font-weight: 500;
+  color: rgb(41, 41, 41);
+  cursor: pointer;
```

Comment:
`cursor: pointer` should be set in a :hover pseudo-class, not in the element itself (we need it only when the element is hovered)